deploy/csi-plugin-scc.yaml

```diff
@@ -0,0 +1,45 @@
+allowHostDirVolumePlugin: true
+allowHostIPC: false
+allowHostNetwork: true
+allowHostPID: false
+allowHostPorts: false
+allowPrivilegeEscalation: true
+allowPrivilegedContainer: true
+allowedCapabilities: []
+apiVersion: security.openshift.io/v1
+defaultAddCapabilities: null
+fsGroup:
+  type: MustRunAs
+groups:
+- system:authenticated
+kind: SecurityContextConstraints
+metadata:
+  annotations:
+    kubernetes.io/description: allow hostpath and host network to be accessible 
+  generation: 1
+  name: csiaccess
+  selfLink: /apis/security.openshift.io/v1/securitycontextconstraints/csiaccess
+readOnlyRootFilesystem: false
+requiredDropCapabilities:
+- KILL
+- MKNOD
+- SETUID
+- SETGID
+runAsUser:
+  type: RunAsAny
+seLinuxContext:
+  type: RunAsAny
+supplementalGroups:
+  type: RunAsAny
+users:
+- system:serviceaccount:spectrum-scale-csi:ibm-spectrum-scale-csi-attacher-sa
```

Comment:
service account names should be changed to ibm-spectrum-scale-csi-attacher i.e without -sa. This is applicable for other sidecar also